.circleci/config.yml

```diff
@@ -127,15 +127,15 @@ jobs:
 
     steps:
       - run:
-        name: Install Node 12 (Required for Yarn)
-        command: |
-          set +e
-          touch $BASH_ENV
-          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
-          echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
-          echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
-          echo 'nvm install v12.13.1' >> $BASH_ENV
-          echo 'nvm alias default v12.13.1' >> $BASH_ENV
+          name: Install Node 12 (Required for Yarn)
+          command: |
+            set +e
+            touch $BASH_ENV
+            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.2/install.sh | bash
+            echo 'export NVM_DIR="$HOME/.nvm"' >> $BASH_ENV
+            echo '[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"' >> $BASH_ENV
+            echo 'nvm install v12.13.1' >> $BASH_ENV
+            echo 'nvm alias default v12.13.1' >> $BASH_ENV
```

Comment:
is this deliberate? I'm pretty sure this step is working in master, there was an indentation issue here a few weeks back but it was fixed.

Reply:
Good callout, looks like this branch started off more behind master then I realized, this change was unnecessary after merging